EIPS/eip-3668.md

```diff
@@ -140,6 +140,26 @@ callData = "0x00112233"
 
 The request URL to query is `https://example.com/gateway/0xaabbccddeeaabbccddeeaabbccddeeaabbccddee/0x00112233.json`.
 
+If the URL template contains the `{data}` substitution parameter, the client MUST send a GET request after replacing the substitution parameters as described above.
```

Comment:
```suggestion
If the URL template contains the `{data}` substitution parameter, the client MUST send a GET request after replacing the substitution parameters as described above. Separating `data` out of the URL is recommended if `data` is of a variable length and/or its inclusion could possibly cause the URL to exceed the 2048 URL character limit.
```
Not tied to the wording, but I think it's worthwhile to call out the limitations that make POST support necessary.